go/tasks/v1/utils/template.go

```diff
@@ -48,9 +47,12 @@ func ReplaceTemplateCommandArgs(ctx context.Context, command []string, args Comm
 	return res, nil
 }
 
-func replaceTemplateCommandArgs(_ context.Context, commandTemplate string, args *CommandLineTemplateArgs) (string, error) {
+func replaceTemplateCommandArgs(ctx context.Context, commandTemplate string, args *CommandLineTemplateArgs) (string, error) {
 	val := inputFileRegex.ReplaceAllString(commandTemplate, args.Input)
 	val = outputRegex.ReplaceAllString(val, args.OutputPrefix)
+	if args.Inputs == nil || args.Inputs.Literals == nil {
```

Comment:
We should probably move this block down a couple of lines... basically:
If we try to reference an input in the template AND there are no inputs in args, then fail...